_pkgdown.yml

```diff
@@ -9,7 +9,10 @@ navbar:
 authors:
   NLBIF:
     href: "https://www.nlbif.nl/move2gbif-gps-zendergegevens-van-dieren-mobiliseren-naar-movebank-en-gbif/"
-    html: "<img src='https://www.nlbif.nl/wp-content/uploads/logo-nlbif.png' width=72 alt='NLBIF logo'>"
+    html: "<img src='https://oscibio.inbo.be/assets/logos/nlbif.png' height='24' alt='NLBIF logo'>"
+  European Union:
+    href: "https://dto-bioflow.eu/"
+    html: "<img src='https://oscibio.inbo.be/assets/logos/funded-by-eu.png' height='24' alt='DTO-BioFlow logo'>"
```

Comment:
I do see the logo of NLBIF, but not the logo of the EU when using `pkgdown::build_articles()`.

![image](https://github.com/user-attachments/assets/ea4d23e0-47b8-479b-8a9d-1d9117979b49)